testsuite/MDAnalysisTests/utils/test_imports.py

```diff
@@ -22,36 +22,42 @@
 from __future__ import absolute_import
 
 import os
+
 import MDAnalysisTests
+import pytest
+
+"""Relative imports are banned in unit testing modules (Issue #189), so run tests to enforce this policy."""
+
+path_to_testing_modules = MDAnalysisTests.__path__[0]
+# Exclusion path relative to MDAnalysisTests
+exclusions = ['/plugins']
```

Comment:
Also add `'/data'` to exclusions – will save a tiny bit in the `os.walk` and makes clear that *nothing* in the data directory should be looked at. (There are no `.py` files in `data` so nothing changes overall anyway.)